src/lib/renderer/renderer.tsx

```diff
@@ -65,29 +58,51 @@ export interface IRenderFormProps {
  * All components receive the `IComponentProps` as props containing the required context to render
  * the component. Components should return a React.ReactElement.
  *
- * @external {CallbacksContext} Expects `CallbackContext` to be available.
- * @external {RenderContext} Expects `RenderContext` to be available.
- * @external {SubmissionContext} Expects `SubmissionContext` to be available.
+ * @external {RenderContext} Provides `RenderContext`.
  */
-export const RenderForm = ({form}: IRenderFormProps): React.ReactElement => {
-  const children =
+export const RenderForm = ({
+  children,
+  configuration,
+  form,
+  formAttrs,
+  initialValues = {},
+  onSubmit,
+}: IRenderFormProps): React.ReactElement => {
+  const childComponents =
     form.components?.map((component: IRendererComponent, i: number) => (
       <RenderComponent key={component.id || i} component={component} />
     )) || null;
-  return <React.Fragment>{children}</React.Fragment>;
+
+  return (
+    <Formik initialValues={initialValues} onSubmit={onSubmit}>
+      {props => {
+        return (
+          <form onSubmit={props.handleSubmit} {...formAttrs}>
```

Comment:
```suggestion
      {({handleSubmit}) => {
        return (
          <form onSubmit={handleSubmit} {...formAttrs}>
```
I find the `props` here confusing as it is convention that your own component props are named like that. The formik docs are inconsistent - I'm seeing examples with destructuring, `formikProps` and `props`.

I think I prefer destructuring in this case.

Reply:
I can see how this could be confusing, and have not strong feelings either. Let's pull this in.

On a sidenote: do we have some kind of convention regarding restructuring? I feel it's a little context dependent to where it adds value.

Reply:
in other places we pretty much always destructure (but only what's used), especially in the component itself, so rather prefer `const Foo = ({bar, baz}) => {...}` than `const Foo = (props) => {...}`)